fixtures/CAPI.ts

```diff
@@ -362,6 +362,15 @@ export const CAPI: CAPIType = {
         AU: { adTargeting: [] },
         INT: { adTargeting: [] },
     },
+    commercialConfiguration: {
```

Comment:
Hey @shtukas my fault for not paying attention on Frontend. But nearly all of this stuff is derivable from the existing model. Is it worth just creating a helper function here to do CAPITYPE -> CommercialConfiguration ?

Reply:
Um........ that makes a lot of sense actually. Let me try.

Reply:
Actually this is just to set the plumbing working. The current attributes of `CommercialConfiguration` as I defined it are only a subset of what we had originally defined would be needed ( see: https://trello.com/c/irRKFsGa/468-identify-all-config-properties-required-for-dotcom-rendering-commercialjs-to-run ). Unless all of that can be extracted from the CAPI data on dotcom-rendering, a helper function cannot be used to recover the information.

Reply:
Hey, thanks. Would it not make sense to use what we already have and only send what isn't already present? Is the plan to expose this on a global object client-side eventually?

Reply:
As discussed IRL, yeah that makes a lot of sense. Less moving parts. Going to close this and focus on deriving `CommercialConfiguration` from the data we pass to DCR.